controllers/admin/AdminTranslationsController.php

```diff
@@ -2772,6 +2772,39 @@ protected function getTinyMCEForMails($iso_lang)
             <script type="text/javascript" src="' . __PS_BASE_URI__ . 'js/admin/tinymce.inc.js"></script>';
     }
 
+    /**
+     * Getting the translated values of each passed subject email
+     * @param array $subject_mail
+     * @param $i18n_dir
+     * @return array
+     */
+    protected function getSubjectMailContent($subject_mail, $i18n_dir)
+    {
+        // Getting the translations depending on theme
+        global $_LANGMAIL;
+        $fileTheme = "$i18n_dir/lang.php";
```

Comment:
Should it be ```
        $file_core = _PS_ROOT_DIR_ . '/mails/' . $isoCode . '/lang.php';
        if (Tools::file_exists_cache($file_core) && empty($_LANGMAIL)) {
            include $file_core;
        }

        $fileTheme = _PS_THEME_DIR_ . 'mails/' . $isoCode . '/lang.php';
        if (Tools::file_exists_cache($fileTheme)) {
            include $fileTheme;
        }
```
Like in Mail.php, maybe you should us `Mail::l` instead?

Reply:
I do not understand when you say «you should use Mail::l instead»...
In the description of my test , I explain than I use Mail::l.
Where should I use Mail::l ?

Reply:
In my memory, I copied/pasted the code from somewhere else, but I can't remember from where …
If my solution is not correct, please treat the bug …

Reply:
This function is doing the job for you by fetching translations.

Reply:
@PierreRambaud I made some research on the PrestaShop core because of the conflict and I don't understand one thing !
You said to me to use Mail::l instead of doing all the stuff I did.
Ok, why not, you're maybe right so I tried to find if it's ok.
But I found a PR : https://github.com/PrestaShop/PrestaShop/pull/19289 which fixes the same kind of problem of mine (not very surprising, I'm not the only one to use translation in module email…)
But in this one, the fix is quite similar of mine and you validated it without using Mail::l.

So, I'll close this PR and bug but for something I waited a year with non very constructive discussions it's very frustrating.